<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>线程</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="btnDiceBegin">
       <property name="text">
        <string>开始</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDiceEnd">
       <property name="text">
        <string>暂停</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnStopThread">
       <property name="text">
        <string>结束线程</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnStartedThread">
       <property name="text">
        <string>启动线程</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清空文本</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>365</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread 状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabPic">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>Pic</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
